test/benchmark.js

```diff
@@ -70,14 +64,24 @@ suite
   .add('rndm', () => {
     rndm(21)
   })
-  .on('cycle', event => {
-    let name = event.target.name.padEnd('async secure-random-string'.length)
-    let hz = formatNumber(event.target.hz.toFixed(0)).padStart(10)
-    if (event.target.name === 'uid') {
-      name = '\nNon-secure:\n' + name
-    }
-    process.stdout.write(
-      `${name}${styleText('bold', hz)}${styleText('dim', ' ops/sec')}\n`
-    )
-  })
-  .run()
+
+const longestNameLength = bench.tasks.reduce((maxLength, task) => Math.max(maxLength, task.name.length), 0)
```

Comment:
This line is not following the project’s code style.

`const` → `let` and be sure that it is follows 80 symbols limit.